almalinux-deploy.sh

```diff
@@ -56,7 +56,7 @@ assert_run_as_root() {
 # Terminates the program if UEFI Secure Boot is enabled
 assert_secureboot_disabled() {
     local -r message='Check Secure Boot disabled'
-    if dmesg | grep -P 'secureboot:\s+Secure\s+boot\s+enabled' 1>/dev/null; then
+    if LC_ALL='en_US.UTF-8' mokutil --sb-state 2>/dev/null | grep -P '^SecureBoot\s+enabled' 1>/dev/null; then
```

Comment:
I think it would be better to use the "C" locale instead of the "en_US.UTF-8" which may not exist.

Reply:
@ezamriy Thanks for the input. Basically it shouldn't really matter as so far `mokutil(1)` isn't localized yet. It's just rather best practice whenever output of a command is being parsed and just in case it gets translation strings in an upcoming version in the future.

Nevertheless, it's changed to locale `C` now.